api/pokemon.tsx

```diff
@@ -1,10 +1,22 @@
-interface Pokemon {
+export interface Sprite {
```

Comment:
Where is it used? I think you could either remove it or change `sprites` type below to `sprite: Sprite` (you will have to add `back_shiny` here)

Reply:
forgot to remove it ;)